includes/Loading/DesignSituation.js

```diff
@@ -0,0 +1,461 @@
+/**
+ * Creates design situation object
+ * @param {Number}  no                      Number of design situation, can be undefined
+ * @param {String}  design_situation_type   Design situation type
+ * @param {Object}  params                  Additional parameters, can be undefined
+ * @param {String}  comment                 Comment, can be undefined
+ * @returns     Created design situation object
+ */
+ function DesignSituation (no,
+    design_situation_type,
+    params,
+    comment) {
+    if (!(design_situation_type in design_situation_types)) {
+        console.log("Design type " + design_situation_type + " does not exist");
+        get_design_situation_types();
+    }
+    this.design_situation = createBaseDesignSituation(no, comment, params);
+    this.design_situation.design_situation_type = design_situation_types[design_situation_type];
+}
+
```

Comment:
Add function to get this.desing_situation as object and also function for this.desgin_situation.no